common/constants.ts

```diff
@@ -5,4 +5,6 @@
 export const PLUGIN_ID = 'aiFlowDashboards';
 
 export const BASE_NODE_API_PATH = '/api/ai_flow';
```

Comment:
Why do we have `api` in the base path?

Reply:
It follows the standard node API naming conventions. Typically `api/<plugin>/<do-something>`. This way frontend plugins won't have name clashing when designing their node APIs

Reply:
Ohh it's for Dashboard plugin. I see.